distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -51,19 +51,20 @@ javax.net.ssl.trustStoreType=jks
 #
 # Global URL Properties
 #
-
-# The httpsPort, httpPort, and port properties are used to advertise the system externally
-# The internalHttpsPort and internalHttpPort properties are the ports that the system will run on
-
+# The httpsPort and httpPort are the ports that the system will run on
 org.codice.ddf.system.protocol=https://
 org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
 org.codice.ddf.system.httpPort=8181
 org.codice.ddf.system.port=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
-org.codice.ddf.system.internalHttpsPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpsPort}
-org.codice.ddf.system.internalHttpPort=_DO_NOT_EXPAND_${org.codice.ddf.system.httpPort}
 org.codice.ddf.system.rootContext=/services
-
+# external.httpsPort and external.httpPort are useful when DDF is behind a proxy and
```

Comment:
✏️ Should we make the comment a little bit more general and say something like `external properties are useful...` instead of being specific to `httpsPort` and `httpPort` since they are all part of the URL presented externally?

Reply:
That's fair, I'll update this.